web/src/main/webapp/WEB-INF/classes/setup/sql/migrate/v3110/migrate-mysql.sql

```diff
@@ -0,0 +1,3 @@
+DROP INDEX idx_metadatafiledownloads_metadataid;
+DROP INDEX idx_metadatafileuploads_metadataid;
+DROP INDEX idx_operationallowed_metadataid;
```

Comment:
Syntax should be updated to:

```
DROP INDEX idx_metadatafiledownloads_metadataid ON MetadataFileDownloads;
DROP INDEX idx_metadatafileuploads_metadataid ON MetadataFileUploads;
DROP INDEX idx_operationallowed_metadataid ON OperationAllowed;
```

Reply:
I was also confuse about the migration mechanism I though both `migrate-default.sql` and `migrate-XXXX.sql` were executed for for db `XXXX`

`migrate-default.sql` is executed, unless specific db file is available. So we can add the drop indexes to the `migrate-default.sql` and create 2 files for mysql (specific syntax) and Oracle (no drop of indexes)

Thanks @juanluisrp for pointing it.

Reply:
@josegar74  Ok, that sounds great - I'll do that.

Reply:
@josegar74 - fixed